app/controllers/application_controller.rb

```diff
@@ -190,6 +192,23 @@ def kick_out_robots
     false
   end
 
+  # Redirect anonymous users to login unless they are looking at allowed pages
+  def redirect_anonymous_users
+    return true if browser.bot? # recognized bots are handled elsewhere
+    return true if verified_user_logged_in?
+
+    store_location
+    redirect_to(account_login_path)
+  end
+
+  private ##########
+
+  def verified_user_logged_in?
+    session_user&.verified?
```

Comment:
Thank you for putting this trivial private helper right next to where it's used!

I see you do this sort of thing frequently and I want to honor that you presumably prefer this style from an aesthetic point of view(?)  My understanding here is that "verified_user_logged_in?" is only scarcely if at all shorter than "session_user&.verified?", therefore this is all about readability.  I agree that "verified_user_logged_in?" is more readable, fair enough.  But the drawback is that I don't know how the helper method actually checks that without locating the helper method and checking.  In this case, it's *right* there, so that's minimal.  But I wonder how it would be different, for example, from in-lining the helper method and adding a brief comment instead:

    return true if session_user&.verified? # verified user logged in?

Super low priority!  Just wanted to engage in a brief conversation with you about this to understand where you're coming from and what your thoughts were.  Thanks!